src/App.js

```diff
@@ -0,0 +1,29 @@
+import logo from './logo.svg';
+import './App.css';
+
+function App() {
+  return (
+    <div className="App">
+      <header className="App-header">
+        <img src={logo} className="App-logo" alt="logo" />
+        <p>
+          Edit
+          {' '}
+          <code>src/App.js</code>
+          {' '}
+          and save to reload.
+        </p>
+        <a
+          className="App-link"
+          href="https://reactjs.org"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          Learn React
+        </a>
+      </header>
+    </div>
+  );
```

Comment:
- [ ] You should create a Greeting component that will be used to interact with the rails backend and present the greeting from the API. Your app is just a dummy app generated by CRA. Ensure your app fetches greeting from the API and renders it on the frontend. Currently, nothing is displayed.

  You will need to set up redux and API fetches for this

  ![image](https://user-images.githubusercontent.com/36542986/226300880-ca66f4e2-3629-43f5-a19d-c47f48aa7dde.png)